utils/build-presets.ini

```diff
@@ -2377,6 +2407,8 @@ skip-test-playgroundsupport
 mixin-preset=
     tensorflow_osx
     mixin_codesigning
+tensorflow-swift-apis
+pythonkit
```

Comment:
macOS toolchain builds (via `utils/build-toolchain-tensorflow --pkg`) currently fail while building `pythonkit`:
```console
...
+ /usr/local/bin/cmake --build /Users/swiftninjas/s4tf/build/buildbot_osx/pythonkit-macosx-x86_64
[1/1][100%][0.341s] Linking Swift shared library PythonKit/libPythonKit.dylib
FAILED: PythonKit/libPythonKit.dylib PythonKit/CMakeFiles/PythonKit.dir/NumpyConversion.swift.o PythonKit/CMakeFiles/PythonKit.dir/Python.swift.o PythonKit/CMakeFiles/PythonKit.dir/PythonLibrary+Symbols.swift.o PythonKit/CMakeFiles/PythonKit.dir/PythonLibrary.swift.o PythonKit/PythonKit.swiftmodule
: && /Users/swiftninjas/s4tf/swift/swift-nightly-install/Library/Developer/Toolchains/swift-tensorflow-DEVELOPMENT-2020-02-20-a.xctoolchain/usr/bin/swiftc -output-file-map PythonKit/CMakeFiles/PythonKit.dir/output-file-map.json -incremental -num-threads 12 -emit-library -o PythonKit/libPythonKit.dylib -module-name PythonKit -module-link-name PythonKit -emit-module -emit-module-path PythonKit/PythonKit.swiftmodule -emit-dependencies -DPythonKit_EXPORTS   /Users/swiftninjas/s4tf/PythonKit/PythonKit/NumpyConversion.swift /Users/swiftninjas/s4tf/PythonKit/PythonKit/Python.swift /Users/swiftninjas/s4tf/PythonKit/PythonKit/PythonLibrary+Symbols.swift /Users/swiftninjas/s4tf/PythonKit/PythonKit/PythonLibrary.swift      && :
/Users/swiftninjas/s4tf/PythonKit/PythonKit/PythonLibrary.swift:18:8: error: cannot load underlying module for 'Darwin'
import Darwin
       ^
<unknown>:0: note: did you forget to set an SDK using -sdk or SDKROOT?
<unknown>:0: note: use "xcrun swiftc" to select the default macOS SDK installed with Xcode
ninja: build stopped: subcommand failed.
Building the standard library for: swift-stdlib-macosx-x86_64
Running Swift tests for: check-swift-all-macosx-x86_64
```

@compnerd: do you have any idea why this occurs?
I'm using CMake 3.15.4 on macOS, upgrading to 3.16.4 now in case that resolves the issue.

Reply:
I tried some things (started clean builds, etc) and can't reproduce the issue above anymore.

Now, macOS toolchain builds are stuck at building `tensorflow`. `ctrl t` suggests that `build-script` is stuck at running `yes`:
```console
Preconfigured Bazel build configs. You can use any of the below by adding "--config=<>" to your build command. See .bazelrc for more details.
	--config=mkl         	# Build with MKL support.
	--config=monolithic  	# Config for mostly static monolithic build.
	--config=ngraph      	# Build with Intel nGraph support.
	--config=numa        	# Build with NUMA support.
	--config=dynamic_kernels	# (Experimental) Build kernels into separate shared objects.
	--config=v2          	# Build TensorFlow 2.x instead of 1.x.
Preconfigured Bazel build configs to DISABLE default on features:
	--config=noaws       	# Disable AWS S3 filesystem support.
	--config=nogcp       	# Disable GCP support.
	--config=nohdfs      	# Disable HDFS support.
	--config=nonccl      	# Disable NVIDIA NCCL support.
Configuration finished
load: 3.02  cmd: yes 94555 running 477.87u 0.39s
load: 3.02  cmd: yes 94555 running 478.69u 0.39s
load: 3.02  cmd: yes 94555 running 478.92u 0.39s
load: 3.02  cmd: yes 94555 running 479.24u 0.39s
load: 3.02  cmd: yes 94555 running 479.43u 0.39s
load: 3.02  cmd: yes 94555 running 479.68u 0.39s
load: 3.02  cmd: yes 94555 running 479.80u 0.39s
load: 3.02  cmd: yes 94555 running 479.90u 0.39s
```

This seems likely to be a bug.